autopilot/sagemaker_autopilot_neo4j_portfolio_churn.ipynb

```diff
@@ -0,0 +1,916 @@
+{
```

Comment:
<p>Branding nit: <code>AWS S3</code> -&gt; <code>Amazon S3</code> or <code>S3</code></p><p> </p><p>Please cite the source of the dataset, including a link and dataset provider.</p><p> </p><p>Please add a disclaimer such as: </p><blockquote><strong>Important</strong>: This example notebook is for demonstrative purposes only. It is not financial advice and should not be relied on as financial or investment advice.</blockquote><p>See another notebook that uses EDGAR data for reference: https://github.com/aws/amazon-sagemaker-examples/blob/main/sagemaker-jumpstart/financial_tabtext_construction/SEC_Retrieval_Summarizer_Scoring.ipynb</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Hi.  I think that's subtly different EDGAR data.  But a similar overall idea.  I've added a note, etc.</p>